app/scripts/directives/fromFile.js

```diff
@@ -33,10 +33,16 @@ angular.module("openshiftConsole")
           $scope.noProjectsCantCreate = true;
         });
 
+        $scope.projectNameTaken = false;
```

Comment:
Shouldn't need to initialize here since the `$watch` below is called immediately and will set it to false.

Reply:
Fixed